src/Testcontainers.Qdrant/Testcontainers.Qdrant.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Are all these really necessary? I would think the following would be enough:

```suggestion
        <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

---

Every other TestContainer module defines these target frameworks, so was sticking to the project convention. I've also added net462 for same reason Milvus module does: https://github.com/testcontainers/testcontainers-dotnet/pull/1131#discussion_r1507712752

---

I think we should stick to the repo convention too 👍. In the end, it does not really matter, right (except for the legacy framework)? We can consider dropping support for all projects. I just kept it because there were some preprocessor directives we used in the past, but I think we have already changed them. I need to double-check.